isstools/widgets/widget_general_info.py

```diff
@@ -19,9 +19,8 @@
 ui_path = pkg_resources.resource_filename('isstools', 'ui/ui_general_info.ui')
```

Comment:
As this `ui_path` is not used below, should it also be commented out or removed?